applications/bci_visualization/streams/kernel_sdk.py

```
@@ -0,0 +1,232 @@
+    ) -> None:
+        if receiver_queue_size <= 0:
+            raise ValueError("receiver_queue_size must be positive")
+        self._receiver_queue_size = int(receiver_queue_size)
+        self._receiver_queue: Queue[np.ndarray] | None = None
+        self._receiver_thread: Thread | None = None
+        self._channels: ChannelInfo | None = None
+        self._good_channels: List[int] = []
+
+    def start(self) -> None:
+        if self._receiver_thread is not None and self._receiver_thread.is_alive():
+            raise RuntimeError("KernelSDKStream is already started")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)